client/my-sites/checkout/cart/header-cart.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { noop, isEmpty } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getAllCartItems } from 'lib/cart-values/cart-items';
+import PopoverCart from './popover-cart';
+
+class HeaderCart extends React.Component {
+	static propTypes = {
+		cart: PropTypes.object,
+		selectedSite: PropTypes.object.isRequired,
+		currentRoute: PropTypes.string,
+	};
+
+	state = {
+		isPopoverCartVisible: false,
+	};
+
+	togglePopoverCart = () => {
+		this.setState( {
+			isPopoverCartVisible: ! this.state.isPopoverCartVisible,
```

Comment:
_Tiny_ nit: it may be worth accepting the current state as an argument to an updater function, just to avoid weird issues where multiple toggle calls might collide. (Ref: https://reactjs.org/docs/react-component.html#setstate)